src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
+        System.out.print("Now you have " + listCount + " task");
+        if(listCount > 1) {
+            System.out.print("s");
+        }
+        System.out.println(" in the list.");
+        return listCount;
+    }
+    public static int addDeadlineToList (Todo[] recordedList, String details, int listCount) {
+        String[] parsedStrings = details.split(" /by ", 2);
+        recordedList[listCount] = new Deadline(parsedStrings[0], parsedStrings[1], "D");
+        System.out.println("Got it. I've added this task:");
+        printTask(recordedList, listCount);
+        listCount++;
+        System.out.print("Now you have " + listCount + " task");
+        if(listCount > 1) {
+            System.out.print("s");
+        }
+        System.out.println(" in the list.");
+        return listCount;
+    }
+    public static int addEventToList (Todo[] recordedList, String details, int listCount) {
+        String[] parsedStrings = details.split(" /at ", 2);
+        recordedList[listCount] = new Event(parsedStrings[0], parsedStrings[1], "E");
+        System.out.println("Got it. I've added this task:");
+        printTask(recordedList, listCount);
+        listCount++;
+        System.out.print("Now you have " + listCount + " task");
+        if(listCount > 1) {
+            System.out.print("s");
+        }
+        System.out.println(" in the list.");
+        return listCount;
+    }
+    public static void completeTask (Todo[] recordedList, int index) {
+        recordedList[index].setAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        printTask(recordedList, (index));
+
+    }
+    public static void bidGoodbye () {
+        printHorizontal();
+        System.out.println("Bye. Hope to see you again soon!");
+        printHorizontal();
+    }
+
+    public static void main(String[] args) {
+        Todo[] recordedList = new Todo[100];
+        int listCount = 0;
+
+        printHeader();
+
+        Scanner input = new Scanner(System.in);
+        String command = input.nextLine();
+        String keyword = extractKeyword(command.trim());
+        String details;
+        if(findCommandLength(command) > 1) {
+            details = extractDetails(command.trim());
+        }
+        else {
+            details = " ";
+        }
+
+        while (!isBye(keyword)) {
+            switch(keyword) {
+            case "todo":
+                printHorizontal();
+                listCount = addTodoToList(recordedList, details, listCount);
+                printHorizontal();
+                break;
+            case "deadline":
+                printHorizontal();
+                listCount = addDeadlineToList(recordedList, details, listCount);
+                printHorizontal();
+                break;
+            case "event":
+                printHorizontal();
+                listCount = addEventToList(recordedList, details, listCount);
+                printHorizontal();
+                break;

```

Comment:
Since todo, deadline and event are all inherited from Task, I believe there's a way to refactor this part to reduce repetition. :)